### ISSUE TITLE
Unable to do anything without logging in

### ISSUE THREAD

**Issue by crutchcorn** [NativeScript/sidekick-feedback#100]
### Did you verify whether this issue has already been reported here?
Yes

### Tell us about the problem.
In previous versions of Sidekick, you were able to do all local functionality without having to login with a Telerik account. I understand there's money to be made, but I just want to be able to get up and running without an account for anything.

### Provide the following version numbers that the issue occurs with:

Version: 1.0.1-v.2017.11.16.1 (latest)
NativeScript CLI version: 3.3.1
CLI extension nativescript-cloud version: 1.0.0
CLI extension nativescript-starter-kits version: 0.3.4

**Comment by tsvetie** [comment]
Hey @crutchcorn,

Thank you for the feedback. Indeed, during the beta period of Sidekick, we required the user to login right before a cloud operation. For the official version, we decided to switch to a required login when you lauch Sidekick. However, this decision was not made based on the desire to monetize on all feature as all local operations continue to be free and we have no plans to change this, but out of a desire to know our customers better. 

To summarize:
- we require that you have an account to use Sidekick
- all local operations are free
- the first 100 cloud builds per month are free
- the pricing options for Sidekick relate only to our cloud services. You can review them here - https://www.nativescript.org/nativescript-sidekick/purchase.

I hope this clarifies the situation a bit.

**Comment by teobugslayer** [comment]
Hey @crutchcorn,

I am the PM of Sidekick. With the login screen at start we are solving two issues:

- Asking for the credentials at inconvenient times. The most extreme example is our [code signing assistance](https://docs.nativescript.org/sidekick/user-guide/code-signing/code-signing-for-ios/code-signing-assistance) feature where people would need to sign up in the middle of the work flow right after they have provided their Apple credentials and before being asked to provide password for the certificate. This really confuses the people - even team members struggled with this.
- We will add new cloud features and for each of them we need to think two work flows - when the users are already logged in and when they are not. Our team has limited resources and removing this complexity allowed us to move faster.

Finally, I really want to reiterate what @tsvetie said:
 - all non-cloud features are completely free
 - we have no immediate plans to change this
 - Sidekick focus is on the cloud services. It enables new scenarios, which are not covered by the CLI, like developing for iOS on Windows

Best regards,
Todor
